.travis.yml

```diff
@@ -0,0 +1,42 @@
+sudo: false
+language: python
+matrix:
+  include:
+    - python: 2.7
+      env: TOXENV=py27-local-slow
+    - python: 2.7
+      env: TOXENV=py27-integ-slow
+    - python: 3.4
+      env: TOXENV=py34-local-slow
+    - python: 3.4
+      env: TOXENV=py34-integ-slow
+    - python: 3.5
+      env: TOXENV=py35-local-slow
+    - python: 3.5
+      env: TOXENV=py35-integ-slow
+    - python: 3.6
+      env: TOXENV=py36-local-slow
+    - python: 3.6
+      env: TOXENV=py36-integ-slow
+#    - python: 3.6
```

Comment:
Any reason not to either note when these are coming back or remove them for now?

Reply:
Fair point. I'll create issues for those and reference them in comments. I'd like to leave the entries, just to remind us to turn them back on when we can.